pixi.toml

```diff
@@ -30,7 +32,10 @@ pytest = ">=8.0.2,<9"
 
 [feature.pytest.tasks]
 test-integration = "pytest tests/integration"
```

Comment:
can we also pass here PIXI_PATH=None or something to ensure that we do not use by accident pixi setup in a path?

Reply:
I now take `pixi` always from a specific path.
That way there's no option for accidents, I also removed the second task as you requested.